tools/Azure.Mcp.Tools.Foundry/src/Models/KnowledgeIndexSchema.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace Azure.Mcp.Tools.Foundry.Models;
+
+/// <summary>
+/// A simplified, AOT-safe representation of a knowledge index schema.
+/// </summary>
+public sealed class KnowledgeIndexSchema
+{
+    [JsonPropertyName("type")]
+    public string? Type { get; set; }
+
+    [JsonPropertyName("id")]
+    public string? Id { get; set; }
+
+    [JsonPropertyName("name")]
+    public string? Name { get; set; }
+
+    [JsonPropertyName("version")]
+    public string? Version { get; set; }
+
+    [JsonPropertyName("description")]
+    public string? Description { get; set; }
+
+    [JsonPropertyName("tags")]
```

Comment:
Ony have jsonpropertyname if diff than property name

Reply:
Seems like we use it everywhere? https://github.com/microsoft/mcp/blob/main/tools/Azure.Mcp.Tools.FunctionApp/src/Models/FunctionAppInfo.cs